orderbook/openapi.yml

```diff
@@ -19,7 +19,6 @@ paths:
           application/json:
             schema:
               $ref: "#/components/schemas/OrderNew"
-  /api/v1/orders:
```

Comment:
This is the error the validation found which https://editor.swagger.io/ also complains about.